src/createRepoRelationship.ts

```diff
@@ -0,0 +1,82 @@
+export async function createRepoRelationship(depsList, mainRepo, j1Client, repoName, missingDeps: string[]) {
```

Comment:
Minor, but this function should probably be named plurally, E.G. `createRepoRelationships` since it creates multiple.